--- ./node_modules/plyr/src/js/captions.js	2025-10-03 18:57:23.246204381 -0700
+++ ./node_modules/plyr/src/js/captions.js	2025-10-03 18:57:38.550835823 -0700
@@ -25,6 +25,8 @@
 const captions = {
   // Setup captions
   setup() {
+    return; // disable due to bug in insertAfter (no element in wrapper)
+
     // Requires UI support
     if (!this.supported.ui) {
       return;

--- ./node_modules/plyr/src/js/media.js	2025-10-03 18:57:55.047516350 -0700
+++ ./node_modules/plyr/src/js/media.js	2025-10-03 18:58:14.504318863 -0700
@@ -35,8 +35,8 @@
         class: this.config.classNames.video,
       });

-      // Wrap the video in a container
-      wrap(this.media, this.elements.wrapper);
+      // Add wrapper to container
+      this.elements.container.appendChild(this.elements.wrapper);

       // Poster image container
       this.elements.poster = createElement('div', {

--- ./node_modules/plyr/src/js/plyr.js	2025-10-03 18:58:36.737235720 -0700
+++ ./node_modules/plyr/src/js/plyr.js	2025-10-03 18:59:10.634633295 -0700
@@ -146,11 +146,6 @@
       return;
     }

-    // Cache original element state for .destroy()
-    const clone = this.media.cloneNode(true);
-    clone.autoplay = false;
-    this.elements.original = clone;
-
     // Set media type based on tag or data attribute
     // Supported: video, audio, vimeo, youtube
     const type = this.media.tagName.toLowerCase();
@@ -270,7 +265,6 @@
     // Wrap media
     if (!is.element(this.elements.container)) {
       this.elements.container = createElement('div');
-      wrap(this.media, this.elements.container);
     }

     // Migrate custom properties from media to container (so they work 😉)
